cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -46,9 +46,23 @@
             'replaces four disclosure forms with two new ones. ' +
             'We have resources to help you comply.'
 } %}
+{% if flag_enabled(request, 'DEBT_COLLECTION_RELEASE') %}
+    {% set consumer_tools_content = {
+        'link': {
+            'text': 'Stories about debt collection',
+            'url':  '/yourstory/debtcollection',
```

Comment:
URL should be the canonical URL, `/consumer-tools/everyone-has-a-story/debt-collection/`